components/chat_ui/common/chat_ui_utils.cc

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/chat_ui/common/chat_ui_utils.h"
+
+#include "base/feature_list.h"
+#include "brave/components/chat_ui/common/features.h"
+
+namespace chat_ui {
+bool IsChatSummarizerEnabled() {
+  return base::FeatureList::IsEnabled(features::kChatSummarizer);
```

Comment:
optional: How about checking if BUILDFLAG for service key and end point are empty too? They seem to be prerequisite for this feature.